git/git-cheatsheet-de.md

```diff
@@ -84,6 +84,14 @@ einen Branch löschen (auch mit ungemergten Änderungen):
 `git branch -D <branchname>`
 
 ## Rebase und Konflikte
+
+einen Branch mit einem anderen rebasen
+
+1. einen Branch A auschecken: `git switch <branchnameA>`
+2. den ausgecheckten Branch A mit einem anderen Branch B rebasen: `git rebase <branchnameB>`
```

Comment:
In Markdown kannst du bei der Nummerierung immer `1.` benutzen - beim Rendern wird das dann fortlaufend. So ist es einfacher, neue Punkte einzufügen.

Reply:
Danke für den Hinweis. Ich habe die Änderung eingebaut.